webdriver/tests/bidi/browsing_context/reload/wait.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+
+async def wait_for_reload(bidi_session, context, wait, expect_timeout):
+    # Ultimately, "interactive" and "complete" should support a timeout argument.
+    # See https://github.com/w3c/webdriver-bidi/issues/188.
+    if expect_timeout:
```

Comment:
NIT: I'd move the timeout assertion in other method or inline

Reply:
This method is reused in all tests, this would introduce some duplication, but OK. DAMP instead of DRY, done.